index.js

```diff
@@ -1 +1,6 @@
-module.exports = require('./lib/primitives');
+var _ = require('lodash');
+
+module.exports = _.extend(
+  require('./lib/primitives'),
+  require('./lib/convert')
+);
```

Comment:
Does this mutate what is exprted from `./lib/primitives`? Is that desired? It almost seems like you need:

```javascript
module.exports = _.extend(
  {},
  require('./lib/primitives'),
  require('./lib/convert')
);
```